sources/packages/backend/apps/queue-consumers/src/services/application/application.service.ts

```diff
@@ -239,4 +248,67 @@ export class ApplicationService {
       })
       .getMany();
   }
+
+  /**
+   * Retrieves second disbursements eligible for a specific notification (likely disbursement
+   * date has passed and the institution has not yet approved the second COE/disbursement).
+   * This method applies several criteria to filter eligible disbursements:
+   * - There's a pending disbursement schedule
+   * - Application status is completed
+   * - No notification of this type has been sent for this assessment
+   * - Disbursement date has passed today
+   * @returns An array of eligible disbursements with relevant details for notification.
+   */
+  async getSecondDisbursementsStillPending(): Promise<
```

Comment:
Minor, after adding [this comment](https://github.com/bcgov/SIMS/pull/4303/files#r1937604189) I thought that it would be better to be explicit in the name of the method that this query is targeting a pending second disbursement when the COE is not approved. Suggestion `getSecondDisbursementsCOEStillPending`.